### PR TITLE
[modefi-staking] Add Strategies - Modefi

### PR DESCRIPTION
**Pull request by Modefi**
Fixes # .

Changes proposed in this pull request:
- Add Strategies
- 
-

### CODE REVIEW

File: src/strategies/modefiMultichain/index.ts [hunk 1]
```diff
@@ -0,0 +1,104 @@
+import { subgraphRequest } from '../../utils';
+import getProvider from '../../utils/provider';
+import strategies from '..';
+
+export const author = 'Modefi';
+export const version = '0.0.1';
+
+const defaultGraphs = {
+  '56': 'https://api.thegraph.com/subgraphs/name/apyvision/block-info',
+  '250': 'https://api.thegraph.com/subgraphs/name/ord786/fantom-blocks'
```
Comment (ChaituVR) [inline src/strategies/modefiMultichain/index.ts:10]: Looks like this is just a copy of `multichain` strategy, is there any reason to make it seperate? I think you can use `multichain` strategy in your space and that strategy can support this networt

https://github.com/snapshot-labs/snapshot.js/blob/master/src/strategies/multichain/examples.json#L41-L43

File: src/strategies/index.ts [hunk 2]
```diff
@@ -201,6 +203,8 @@ export default {
   brightid,
   'inverse-xinv': inverseXINV,
   modefi,
+  'modefiStaking': modefiStaking,
```
Comment (ChaituVR) [inline src/strategies/index.ts:206]: The strategy name should be like this.. can u plz rename the folder name too.. 

```suggestion
  'modefi-staking': modefiStaking,
```

**Review by ChaituVR — CHANGES_REQUESTED** [review]

**Comment by Modefi** [comment]
Strategy updated

**Review by ChaituVR — COMMENTED** [review]
LGTM

**Review by bonustrack — APPROVED** [review]

**Pull request merged**